CMSgov/design-system
# [WNMGDS-1121] icon css refs cleanup

## Summary
Updated components to use inline SVGs instead of svgs as css background images

_Please note: USA Banner will be in a separate PR because it required some refactor & talking to Patrick, we will not be updating the `Dropdown` component because it's more of a look and feel than an icon_

### Removed
Removing references to .svg files in css of the following components
* Alert
* Accordion
* Vertical Nav
* Step List

## How to test
Run doc site locally and confirm that changed components look like their original versions

See demo sites at:
* Storybook - http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1121/icon-css-refs-cleanup/core/storybook/?path=/story/components-accordion--controlled
* Doc site - http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1121/icon-css-refs-cleanup/core/
